sdk/Sdk/Targets/Microsoft.Azure.Functions.Worker.Sdk.targets

```
@@ -34,6 +34,7 @@ WARNING:  DO NOT MODIFY this file unless you are knowledgeable about MSBuild and
         <MSBuildFunctionsTargetsPath>$(MSBuildExtensionsPath)\Microsoft\VisualStudio\Managed.Functions\</MSBuildFunctionsTargetsPath>
       
         <FunctionsEnableWorkerIndexing Condition="$(FunctionsEnableWorkerIndexing) == ''">false</FunctionsEnableWorkerIndexing>
+        <FunctionsDisableFuncMetadataSourceGen Condition="$(FunctionsMetadataSourceGen_Enabled) == ''">true</FunctionsDisableFuncMetadataSourceGen>
```

Need to update this properter name:

```suggestion
        <FunctionsMetadataSourceGen_Enabled Condition="$(FunctionsMetadataSourceGen_Enabled) == ''">true</FunctionsMetadataSourceGen_Enabled>
```